### PR TITLE
Add custom event

### PR DESCRIPTION
**Pull request by idegtiarov**
Add custom event included tutor url to the adaptive problem chosen by Edx Adapt, after user enrolled successfully. Event is used to customize navigation next button on the EdxAdapt enrolling page.

### CODE REVIEW

File: edxadapt/static/js/src/edxadapt.js [hunk 1]
```diff
@@ -68,9 +68,51 @@ function EdxAdaptXBlock(runtime, element) {
...
+            document.dispatchEvent(event);
+        } else {
+            console.log("Failed to get tutor_url to the first adaptive problem");
+        }
+    }
+
+   function getFirstAdaptiveProblem(data) {
+        // Get url to the adaptive problem which is chosen by EdxAdapt
+        if (data) {
+            return createEvent(data);
```
Comment (z4y4ts) [inline edxadapt/static/js/src/edxadapt.js]: `createEvent` returns nothing. So you probably don't need `return` here either

File: edxadapt/static/js/src/edxadapt.js [hunk 2]
```diff
@@ -68,9 +68,51 @@ function EdxAdaptXBlock(runtime, element) {
             console.log("Failed to register student in EdxAdapt", jqXHR.responseText);
         });
     }
-    function setStatusOk() {
+
+    function createEvent(data) {
```
Comment (z4y4ts) [inline edxadapt/static/js/src/edxadapt.js]: Change to more descriptive name.

**Review by z4y4ts — COMMENTED** [review]
Please use JSDoc format for new functions.

File: edxadapt/static/js/src/edxadapt.js (hunk 2 above)
Comment (idegtiarov) [reply edxadapt/static/js/src/edxadapt.js]: done

File: edxadapt/static/js/src/edxadapt.js (hunk 1 above)
Comment (idegtiarov) [reply edxadapt/static/js/src/edxadapt.js]: done

**Pull request merged**